src/databaseTest/java/net/sf/jabref/shared/DBMSConnectorTest.java

```diff
@@ -26,12 +27,9 @@
     public void testGetNewConnection() throws ClassNotFoundException, SQLException {
         DBMSConnectionProperties properties = TestConnector.getConnectionProperties(dbmsType);
 
-        try {
+        AssertUtil.assertDoesNotThrow(() -> {
```

Comment:
No idea why this is needed. If an exception is thrown than the test fails automatically?!